Automattic/wp-calypso
# Theme Tiers: Remove feature flag

## Proposed Changes

Removes the `themes/tiers` feature flag since the new theme tiers system has been live for a while now without issues, so we no longer need the previous behavior.

## Testing Instructions

- Use the Calypso live link below
- Check the Design Picker and/or the Theme Showcase
- Make sure there are no regressions by checking that everything works like in production
